README.md

```diff
@@ -1,6 +1,29 @@
+![](./media/Standard_4_Person_Clover_Animation.gif)
+
 Madison Area Jugglers pattern book
 ==================================
 
 This is the working home of the MAJ's pattern book; you can access the
 latest *official* version in PDF format from [our web
 site](http://madjugglers.com/Madison%20Pattern%20Book).
+
+Dependencies
```

Comment:
It would be nice to get a list of all modules, plugins, and packages used with R and (La)Tex during the build.  Is this possible?